### ISSUE TITLE
Capture `patchwork` output

### ISSUE THREAD

**Issue by TylerGrantSmith** [thebioengineer/camcorder#12]
This should follow a similar principle.
``` r
library(ggplot2)
library(patchwork)
g <- ggplot()
h <- ggplot()
dir <- fs::file_temp()
camcorder::gg_record(dir)
g / h
```

![](https://i.imgur.com/x7owRqD.png)

``` r
### errors because no print captured
camcorder::gg_playback()
#> Error in normalizePath(path.expand(path), winslash, mustWork): path[1]=NA
```

<sup>Created on 2020-09-11 by the [reprex package](https://reprex.tidyverse.org) (v0.3.0)</sup>

**Comment by thebioengineer** [comment]
Thanks for the comment! This is currently addressed in #9 and will be merged in soon!

**Comment by thebioengineer** [comment]
resolved with #9 - many months after I promised it would be merged "soon" 😂
